File: May-18/12-05-18/python/01-Insertion-sort.py
n = int(input())
ar = [int(x) for x in input().split()]
pos = []

for i in range(0, n):
    pos.append(ar[i])

ar.sort()

for i in range(0, n):
    for j in range(0, n):
        if(pos[i] == ar[j]):
            print(j+1, end = ' ')
        

File: May-18/12-05-18/java/01-Insertion-sort.java

import java.util.*;


class TestClass {
    public static void main(String args[] ) throws Exception {
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        int ar[] = new int[n];
        int pos[] = new int[n];
        
        for(int i = 0; i < n; i++)
        {
            ar[i] = input.nextInt();
            pos[i] = ar[i];
        }
        
        Arrays.sort(ar);
        
        for(int i = 0; i < n; i++)
        {
            for(int j = 0; j < n; j++)
            {
                if(pos[i] == ar[j])
                {
                    System.out.print( (j+1) + " ");
                }
            }
        }
    }
}
